src/mod/endpoints/mod_sofia/rtp.c

```diff
@@ -479,7 +609,7 @@ static switch_status_t channel_receive_event(switch_core_session_t *session, swi
         if (compare_var(event, channel, kCODEC) ||
             compare_var(event, channel, kPTIME) ||
             compare_var(event, channel, kPT) ||
-	    compare_var(event, channel, kRATE)) {
+	        compare_var(event, channel, kRATE)) {
```

Comment:
this whole file seems to be a combination of tabs and spaces :) but i'd say that the intention was to use spaces for indent

Reply:
Yes the whole file is half spaces, half tab. Fixing the spaces will just confuse the reviewer. Better create a separate PR or do this after merging the latest FS version